crates/iota/src/validator_commands.rs

```diff
@@ -107,8 +107,8 @@ pub enum IotaValidatorCommand {
     DisplayMetadata {
         #[arg(name = "validator-address")]
         validator_address: Option<IotaAddress>,
-        #[arg(name = "json", long)]
-        json: Option<bool>,
+        #[arg(long, global = true)]
+        json: bool,
```

Comment:
I know it was already there but there is already a global json flag at the ValidatorCommand level, can we not pass it down to the execute function? Otherwise it's a bit confusing to be able to do `iota validator --json list --json`

Reply:
Yeah

Reply:
Done https://github.com/iotaledger/iota/pull/8187/commits/4d29170c8958f139071d1091c86baed04f603cf2